Sources/SourceControl/Repository.swift

```diff
@@ -118,6 +118,12 @@ public protocol RepositoryProvider {
     ///   - path: The location of the repository on disk, at which the repository
     ///     has previously been created via `cloneCheckout`.
     func openCheckout(at path: AbsolutePath) throws -> WorkingCheckout
+
+    /// Copies the repository at path `from` to path `to`.
+    /// - Parameters:
+    ///   - sourcePath: the source path.
+    ///   - destinationPath: the destination  path.
+    func copy(from sourcePath: AbsolutePath, to destinationPath: AbsolutePath) throws
```

Comment:
Since the protocol is being changed, could we add an extension here with a default implementation of the new protocol method (to for example just to a regular copy in the file system)? Otherwise clients will immediately fail to compile.

Reply:
done

Reply:
Thanks!